.github/workflows/release.yaml

```diff
@@ -23,23 +23,19 @@ jobs:
       fail-fast: false
       matrix:
         runs-on: [[ubuntu-22.04], [self-hosted, jammy, ARM64]]
```

Comment:
Out of scope here, but if we're simply downloading the artefacts, and not building the snap, this means we can run this job once on ubuntu-22.04 - iterate over the available snap file artefacts, uploading each one.

Reply:
I don't think it's out of scope. It's a good observation and I changed to run just on amd